packages/sit-onyx/src/components/OnyxErrorTooltip/OnyxErrorTooltip.ct.tsx

```diff
@@ -0,0 +1,97 @@
+import { expect, test } from "../../playwright/a11y";
+import OnyxErrorTooltip from "./OnyxErrorTooltip.vue";
+
+test("should render without error-tooltip", async ({ mount, makeAxeBuilder }) => {
+  // ARRANGE
+  const component = await mount(<OnyxErrorTooltip>Dummy content</OnyxErrorTooltip>);
+
+  // ACT
+  await component.getByText("Dummy content").hover();
+  // ASSERT
+  await expect(component.getByRole("tooltip")).toHaveCount(0);
```

Comment:
```suggestion
  await expect(component.getByRole("tooltip")).toBeHidden();
```

Isn't this more appropriate?

Reply:
for me, "Element is not visible" sounded like it exists but is visually hidden, I thought to have count sounds more like "does not exist". 
¯\_(ツ)_/¯

I updated it toBeHidden if we use that more frequently